rust/rsc/src/rsc/main.rs

```diff
@@ -61,21 +62,38 @@ struct ServerOptions {
 async fn activate_stores(
     conn: Arc<DatabaseConnection>,
 ) -> HashMap<Uuid, Arc<dyn blob::DebugBlobStore + Sync + Send>> {
-    let stores = match blob_store_service::fetch_local_blob_stores(&conn).await {
+    let mut active_stores: HashMap<Uuid, Arc<dyn blob::DebugBlobStore + Sync + Send>> =
+        HashMap::new();
+
+    // --- Activate Test Blob Stores  ---
+    let test_stores = match blob_store_service::fetch_test_blob_stores(&conn).await {
```

Comment:
I'll think about it, maybe a conditional compile or something, but this is essentially skipped in production since no results would be returned from the DB. 

Also fortunately this is part of startup so we don't pay a recurring cost during runtime.